`!=` operator is not supported in queries

Firestore supports `!=` operators now: https://firebase.google.com/docs/firestore/query-data/queries#query_operators

I get the following error (which goes away if I use the `==` operator instead). 
```
doc_snapshots = [doc_snapshot for doc_snapshot in doc_snapshots
>                    if compare(doc_snapshot._get_by_field_path(field), value)]
E   TypeError: 'NoneType' object is not callable
```

I believe a simple condition just needs to be added here in `compare_func`: https://github.com/mdowds/python-mock-firestore/blob/master/mockfirestore/query.py#L122

--- a/mockfirestore/query.py
+++ b/mockfirestore/query.py
@@ -121,6 +121,8 @@ class Query:
     def _compare_func(self, op: str) -> Callable[[T, T], bool]:
         if op == '==':
             return lambda x, y: x == y
+        elif op == '!=':
+            return lambda x, y: x != y
         elif op == '<':
             return lambda x, y: x < y
         elif op == '<=':

--- a/tests/test_collection_reference.py
+++ b/tests/test_collection_reference.py
@@ -81,6 +81,16 @@ class TestCollectionReference(TestCase):
         docs = list(fs.collection('foo').where('valid', '==', True).stream())
         self.assertEqual({'valid': True}, docs[0].to_dict())
 
+    def test_collection_whereNotEquals(self):
+        fs = MockFirestore()
+        fs._data = {'foo': {
+            'first': {'count': 1},
+            'second': {'count': 5}
+        }}
+
+        docs = list(fs.collection('foo').where('count', '!=', 1).stream())
+        self.assertEqual({'count': 5}, docs[0].to_dict())
+
     def test_collection_whereLessThan(self):
         fs = MockFirestore()
         fs._data = {'foo': {

